domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/education/PreviousQualifications.kt

```diff
@@ -24,20 +26,15 @@ data class PreviousQualifications(
   val lastUpdatedAtPrison: String,
 )
 
-enum class EducationLevel {
-  PRIMARY_SCHOOL,
-  SECONDARY_SCHOOL_LEFT_BEFORE_TAKING_EXAMS,
-  SECONDARY_SCHOOL_TOOK_EXAMS,
-  FURTHER_EDUCATION_COLLEGE,
-  UNDERGRADUATE_DEGREE_AT_UNIVERSITY,
-  POSTGRADUATE_DEGREE_AT_UNIVERSITY,
-  NOT_SURE,
-}
-
 data class Qualification(
+  val reference: UUID?,
```

Comment:
As well as refactoring the domain, we do need these extra fields in the `Qualification` domain object